src/mlpack/methods/range_search/rs_model_impl.hpp

```diff
@@ -271,11 +271,12 @@ void MonoSearchVisitor::operator()(RSType* rs) const
 }
 
 //! Save parameters for bichromatic range search.
-inline BiSearchVisitor::BiSearchVisitor(const arma::mat& querySet,
-                                        const math::Range& range,
-                                        std::vector<std::vector<size_t>>& neighbors,
-                                        std::vector<std::vector<double>>& distances,
-                                        const size_t leafSize):
+inline BiSearchVisitor::
+BiSearchVisitor(const arma::mat& querySet,
```

Comment:
This looks a little bit strange to me, what do you think about:

```c++
inline BiSearchVisitor::BiSearchVisitor(
    const arma::mat& querySet,
    const math::Range& range,
    std::vector<std::vector<size_t>>& neighbors,
    std::vector<std::vector<double>>& distances,
    const size_t leafSize):
```

Reply:
It looks so good to me. :) Thanks, I'll edit it.